routes/serverSetup.js

```diff
@@ -24,7 +24,6 @@ const path = require('path')
             }
           });
         } catch (err) {
-          console.log(err, routePathIn);
           throw new Error(routeErrorText);
```

Comment:
Looking at this it would be nice to output the routePathIn as part of the Error that is bing thrown since we won't be writing it to console anymore. Do you mind tweaking the routeErrorText in the Error to include it? Inlining and using a template literal is probably the best bet here. If the routeErrorText variable isn't used anywhere else it can be removed at the same time.